src/abstractdataframe/reshape.jl

```diff
@@ -38,9 +38,14 @@ melt(df::AbstractDataFrame, id_vars)
   stacking, a normal column indexing type; for `stack` defaults to all
   variables that are not `measure_vars`
 
+
 If neither `measure_vars` or `id_vars` are given, `measure_vars`
 defaults to all floating point columns.
 
+The names of the resulting "stacked" columns can be controlled with keyword
```

Comment:
Should use a bullet for each argument. You can move the paragraph above to the `measure_vars` bullet so that it's not out of place.